WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -2366,7 +2366,8 @@ private static bool IsHandle(IRedType? irb, bool weakHandleOnly)
         }
 
         var gt = propertyType.GetGenericTypeDefinition();
-        return (weakHandleOnly && gt == typeof(CWeakHandle<>)) || gt == typeof(CHandle<>);
+        return (weakHandleOnly && gt == typeof(CWeakHandle<>)) || gt == typeof(CWeakHandle<>) ||
```

Comment:
This is missing a !weakHandleOnly in the right side of the OR

Reply:
If it's not weakhandleonly, it doesn't matter :)

Reply:
but it's an OR. The whole conditional will be true if the type is a weakhandle even if weakHandleOnly is true.